CRM/Financial/BAO/Payment.php

```diff
@@ -36,16 +36,14 @@ class CRM_Financial_BAO_Payment {
    *   - contribution_id
    *   - total_amount
    *   - line_item
+   * @param bool $disableActionsOnCompleteOrder Disble membership, participant processing when the payment completes the order
+   *   Note this is currently only set by Payment.Create in APIv4
```

Comment:
Can we go further & say this is currently only supported when set by the payment api

Reply:
And it may change!